scenes/scene.yaml

```diff
@@ -6,10 +6,10 @@ fonts:
         url: https://fonts.gstatic.com/s/montserrat/v7/zhcz-_WihjSQC0oHJ9TCYL3hpw3pgy2gAi-Ip7WPMi0.woff
     Open Sans:
         - weight: 300
-          url: fonts/open sans-300normal.ttf
+          url: https://github.com/tangrams/bubble-wrap/raw/gh-pages/fonts/OpenSans-Regular.woff
```

Comment:
We have been trying to not include fonts in tangram es repo..

We can probably point this to some proper url, but this is a just a copy of what tangram JS has in its demo scene.